### ISSUE TITLE
Not Executing Upgrade Searches

### ISSUE THREAD

**Issue by TurtleHead36** [Feramance/qBitrr#138]
SUMMARY:
Hey Feramance I hope all is well developing qBitrr+, Been using this tool for quite a while and its been working great until recently. I'm not sure why it's happened but qBitrr seems to only be managing stalled download in qBittorrent and nothing else. I have it configured to upgrade and search for unmet scores, but it just updates the database and spams "No downloads sleeping for x time" with no other search attempts.

NOTES:
I am currently using the latest version of qBitrr as of this post. I am currently using the Windows .exe (Window 10) version of the tool. I also have tried it on my Linux server using docker but it seems to have the exact same behavior. I'm hoping it's a config issue but as far as I'm aware its proper due to its past perfect behavior. Thanks!

Here's everything I think you might need, let me know if you need anything else!
[qBitrr Issue.zip](https://github.com/user-attachments/files/17964080/qBitrr.Issue.zip)

**Comment by Feramance** [comment]
Please let qbitrr run for 10-15 mins then send a fresh batch of logs, not enough data to work with

**Comment by TurtleHead36** [comment]
Here's a set of logs that ran for 16 minutes. 
[qBitrr 16m.zip](https://github.com/user-attachments/files/18008720/qBitrr.16m.zip)

**Comment by Feramance** [comment]
Checking

**Comment by TurtleHead36** [comment]
Hey checking in, Those logs provide what you need or do I need to run it longer?

**Comment by Feramance** [comment]
Yes and no, though not likely due to anything you did. Could you maybe try the disabling the request search and let me know if things work that way?

**Comment by TurtleHead36** [comment]
I disabled the "Manage Overseerr Requests" and now it appears to be upgrading and applying my CF scores. Thank you so much for the help. It certainly has something to do with Ombi/Overseerrs integration.

**Comment by Feramance** [comment]
I'm working on a fix to rectify this still as yes it does

**Comment by Feramance** [comment]
Latest update should have fixed this. Reopen if not and provide logs please
